roles/scylla-collectd-client/tasks/main.yaml

```diff
@@ -1,5 +1,17 @@
+- name: Gather facts
+  action: ec2_facts
+
+- set_fact: node="scylla-{{ansible_hostname }}"
```

Comment:
On Thu, Nov 5, 2015 at 2:27 PM, Tzach Livyatan <notifications@github.com>
wrote:

> In roles/scylla-collectd-client/tasks/main.yaml
> <https://github.com/scylladb/cassandra-test-and-deploy/pull/23#discussion_r44010459>
> :
>
> > @@ -1,5 +1,17 @@
> > +- name: Gather facts
> > +  action: ec2_facts
> > +
> > +- set_fact: node="scylla-{{ansible_hostname }}"
>
> There no problem using in EC2 information, like the name (DB), server
> (scylla), user (tzach), external and internal IP in any combination.
>
> For example, we can use:
> "DB-scylla-tzach-external-ip"
>

That's going to be a huge string. Defeats a bit the point of using it, no?



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scylladb/cassandra-test-and-deploy/pull/23/files#r44010459>
> .
>